### ISSUE TITLE
MVP - integrate HOLON model into backend (from model perspective)

### ISSUE THREAD

**Issue by noracato** [ZEnMo/Holon-webapp#193]
Based on issue #118, let's track all tasks we still have to finish before we consider being doen for the MVP from the model-side.

## Steps left for integration MVP

To close issue #160 (economics)

- [x] Supply grid connection input json (as list/dict) to module
- [x] Supply correct AL output to module
- [x] Extract function from `post`
- [ ] ~~What to do with separate categories?~~ out of scope
- [ ] ~~Do some basic/hacky caching on ETM results to save calculation time?~~ out of scope

---

To close issue #168 (upscaling)

- [x] Supply correct AL output to module
- [x] Supply user slider settings to module
- [x] Update configs with correct AL-output and slider keys
- [ ] ~~If time left: combine setting and getting ETM settings to save on runtime~~ out of scope

---

To close issue #161 (dataclasses)

- [x] Fix the issue with making the `cloudclient` `pip` 
installable ZEnMo/HOLON-cloudclient/issues/12
- [x] https://github.com/ZEnMo/HOLON-cloudclient/issues/9
- [x] https://github.com/ZEnMo/HOLON-cloudclient/issues/4
- [x] Update datamodel to latest model version

---

To close issue #127 (sliders)

- [x] Datamodel that implements the mapping table to convert sliders input to changes in the config `json`
- [x] Utility function that loops through the `json` to update the desired `attribute` to the new value based on the mapping table

---

Other things to keep in mind

- [ ] Track technical debt in #175

**Comment by thesethtruth** [comment]
![image](https://user-images.githubusercontent.com/78690362/203526353-d2610079-a646-471a-b484-028700c79a8d.png)
